pontoon/base/views.py

```diff
@@ -158,9 +159,51 @@ def projects(request, template='projects.html'):
     return render(request, template, data)
 
 
-def translate(request, locale, slug, part=None, template='translate.html'):
+def locale_project(request, locale, slug):
+    """Locale-project overview."""
+
+    # Redirect to translate view
+    if request.GET.get('q'):
```

Comment:
Yes, and we were using the same URL for search as locale_project uses. I'll add a '/search/' piece to it now.